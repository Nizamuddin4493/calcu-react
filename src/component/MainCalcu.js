import React from 'react';
import CalcoContent from './CalcoContent';
import Calculator from './Calculator';

const MainCalcu = () => (
  <>
    <CalcoContent />
    <Calculator />
  </>
);

export default MainCalcu;
