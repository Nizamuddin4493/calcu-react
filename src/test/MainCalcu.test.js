import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import MainCalcu from '../component/MainCalcu';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><MainCalcu /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
