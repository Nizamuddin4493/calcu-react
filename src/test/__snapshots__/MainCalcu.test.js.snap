// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div>
    <h1
      className="ldsc"
    >
      Lets do some Math...
    </h1>
  </div>,
  <div
    className="calculator"
  >
    <div
      className="result"
    >
      0
    </div>
    <div
      className="buttons"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
         =
      </button>
    </div>
  </div>,
]
`;
